test-requirements.txt

```diff
@@ -3,8 +3,9 @@
 # process, which may cause wedges in the gate later.
 hacking>=1.1.0,<1.2.0 # Apache-2.0
 
--e git+https://opendev.org/openstack/neutron.git@stable/rocky#egg=neutron
+neutron
 
+contextlib2>=0.5.4
```

Comment:
I'm wondering where this is from? My understanding is that this is based on this list,
https://github.com/openstack/neutron/blob/stable/stein/test-requirements.txt
but I didn't find contextlib2 there so.....

Reply:
I had to add this - the python35 job was failing. I've seen references to it not being installed, even though it's considered a standard library. Adding this fixed the python35 job.